paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodVerticalLayoutInteractor.kt

```diff
@@ -412,7 +412,11 @@ internal class DefaultPaymentMethodVerticalLayoutInteractor(
                 }
             }
             is ViewAction.EditPaymentMethod -> {
-                onEditPaymentMethod(viewAction.savedPaymentMethod)
+                if (FeatureFlags.useNewUpdateCardScreen.isEnabled) {
```

Comment:
we should be able to completely consolidate the EditPaymentMethod action with the OnManageOneSavedPaymentMethod action when I remove this feature flag, but for now, this is the simplest way to ensure everything works with the feature flag enabled!